shift-react-app/src/shiftReactApp.ts

```diff
@@ -7,7 +7,7 @@ import {
   ignoreShift,
 } from './steps';
 
-const steps = [
+const steps: Array<(() => Promise<string>) | (() => string) | (() => void)> = [
```

Comment:
Why is this suddenly required?

Reply:
It looks like typescript inferred that the array is of functions of the form `() => void` based on the first function in the array, and allows assigning `() => string` to a `() => void`.